test/flutter_redux_test.dart

```diff
@@ -324,7 +324,6 @@ void main() {
       final widget = () => StoreProvider<String>(
             store: store,
             child: StoreConnector<String, String>(
-              onWillChange: (_) => states.add(BuildState.before),
```

Comment:
This line is actually the heart of this test and should not be removed, but rather updated to the new function signature.